data_safe_haven/functions/__init__.py

```diff
@@ -11,14 +6,13 @@
     sanitise_sre_name,
     seeded_uuid,
     sha256hash,
+    time_as_string,
     truncate_tokens,
 )
 
 __all__ = [
-    "allowed_dns_lookups",
     "alphanumeric",
     "b64encode",
-    "ordered_private_dns_zones",
```

Comment:
Wondering if it would make sense to make `functions` export categories like `strings`.

```python

from data_safe_haven.functions import strings


strings.b64encode(...)
```

A bit like with the validators package.

Reply:
Captured in #1886